sdk/attestation/azure-security-attestation/README.md

```diff
@@ -116,8 +116,7 @@ authentication, the documentation for that API will reflect that the attestation
 
 To interact with the authenticated APIs supported by the Azure Attestation service, your client must present an Azure Active Directory bearer token to the service.
 
-The simplest way of providing a bearer token is to use the  `ClientSecretCredential` authentication method by providing client secret credentials is being used in this
```

Comment:
Would it make sense to include a comment about the drawbacks of DAC in production?

Reply:
#5205